get_lines.py

```
@@ -0,0 +1,21 @@
+def count_lines(filepath):
+    try:
+        with open(filepath, 'r') as f:
+            lines = f.readlines()
+            # Simple heuristic for executable lines
+            executable = 0
+            for line in lines:
+                l = line.strip()
+                if l and not l.startswith('//') and not l.startswith('import ') and not l.startswith('export ') and l != '{' and l != '}':
+                    executable += 1
+            return executable
+    except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)